protobuf/scalapb/scalapb.proto

```diff
@@ -291,6 +291,18 @@ message EnumOptions {
   // must be present.
   optional string type = 3;
 
+  // Custom annotations to add to the generated case object.
+  repeated string annotations_for_trait = 4;
+
+  // Custom annotations to add to the generated case object.
+  repeated string annotations_for_value = 5;
+
+  // Custom annotations to add to the generated Unrecognised case class.
+  repeated string annotations_for_unrecognized = 6;
+
+  // If true, the generated Unrecognized case class will be private.
```

Comment:
I prefer to avoid having specific presets for consistency with the other annotations. There are different types of private in Scala (private and package-private) and different users might want different things.

Reply:
I'm moving this conversation to the main thread